apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/HospiceClaimTransformer.java

```diff
@@ -86,7 +88,6 @@ private static ExplanationOfBenefit transformClaim(HospiceClaim claimGroup) {
         Optional.of(claimGroup.getUtilizationDayCount()));
 
     if (claimGroup.getHospicePeriodCount().isPresent()) {
-      // TODO should this be benefitBalance?
```

Comment:
From what I can see that field should not be mapped as the benefitBalance. Benefit Balance is used  for the actual Balance by Benefit Category overall. Not what was used per claim.